api/v1/api.go

```diff
@@ -62,3 +66,41 @@ func (s *apiV1) RegisterRoutes(e *echo.Echo) {
 	// /constraint-labels
 	e.GET("/constraint-labels", s.handleGetConstraintLabels)
 }
+func GetTenantId(c echo.Context) int {
+	return int(c.Get(TENANT_CONTEXT).(AuthContext).TenantID)
+}
+
+func CreateErrorResponseEnvelop(c echo.Context, err string) ResponseEnvelope {
+	return ResponseEnvelope{
+		RequestUUID:        c.Response().Header().Get(echo.HeaderXRequestID),
+		ResponseTime:       time.Now(),
+		ResponseStateEpoch: time.Now().UTC().UnixMilli(),
```

Comment:
@jcace what did you intend for this to be?

Reply:
I'm not sure, @ribasushi  what is this?

Reply:
It should be the current Filecoin epoch (which is just `uint` not a `Time` value.  It should look like a number on this thing: https://www.epochclock.io/

You can calculate it like 
```go
func UnixToFilEpoch(unixEpoch int64) int64 {
	return int64(math.Floor(float64(unixEpoch-FILECOIN_GENESIS_UNIX_EPOCH) / 30))
}
```

Where 
`FILECOIN_GENESIS_UNIX_EPOCH = 1598306400`

@ribasushi please chime in if there's any issues with that approach (i.e, should we query an actual chain endpoint for it vs. computing here?) This is how we did it in Estuary

Reply:
This is correct, only thing I would change is not using floting point math: integer division is enough. Some SQL examples:

```
-- epoch_from_ts
CREATE OR REPLACE
  FUNCTION epoch_from_ts(TIMESTAMP WITH TIME ZONE) RETURNS INTEGER
LANGUAGE sql PARALLEL SAFE IMMUTABLE STRICT AS $$

  SELECT ( EXTRACT( EPOCH FROM $1 )::BIGINT - 1598306400 ) / 30

$$;

-- ts_from_epoch
CREATE OR REPLACE
  FUNCTION ts_from_epoch(INTEGER) RETURNS TIMESTAMP WITH TIME ZONE
LANGUAGE sql PARALLEL SAFE IMMUTABLE STRICT AS $$

  SELECT TIMEZONE( 'UTC', TO_TIMESTAMP( $1 * 30::BIGINT + 1598306400 ) )

$$;

-- round_down_epoch
-- 2h +/- because network started at 22:00 UTC
CREATE OR REPLACE
  FUNCTION round_down_epoch(INTEGER) RETURNS INTEGER
LANGUAGE sql PARALLEL SAFE IMMUTABLE STRICT AS $$

  SELECT ($1 - 240) / 2880 * 2880 + 240

$$;
```